Cargo.toml

```diff
@@ -1,12 +1,27 @@
 [workspace]
-
 members = [
     "relations",
     "snark",
 ]
 
 resolver = "2"
 
+[workspace.package]
+version = "0.5.1"
```

Comment:
Shouldn't this be "0.5.0"?

Reply:
I had an accidental abort during the release of 0.5 for this crate, and it ended in a weird half-published state on crates.io